tests/nested/manual/preseed/task.yaml

```diff
@@ -11,111 +11,116 @@ environment:
   IMAGE_MOUNTPOINT: /mnt/cloudimg
 
 prepare: |
-  #shellcheck source=tests/lib/nested.sh
-  . "$TESTSLIB/nested.sh"
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
 
-  #shellcheck source=tests/lib/snaps.sh
-  . "$TESTSLIB"/snaps.sh
+    #shellcheck source=tests/lib/nested-tool.sh
+    . "$TESTSLIB/nested-tool.sh"
 
-  #shellcheck source=tests/lib/preseed.sh
-  . "$TESTSLIB/preseed.sh"
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
 
-  # install the snapd deb from spread so we are using the same version to 
-  # validate the seed as well as call preseed, etc. 
-  dpkg -i "$SPREAD_PATH"/../snapd_*.deb
+    #shellcheck source=tests/lib/preseed.sh
+    . "$TESTSLIB/preseed.sh"
 
-  # create a VM and mount a cloud image
-  nested_create_classic_vm
-  mkdir -p "$IMAGE_MOUNTPOINT"
-  IMAGE_NAME=$(nested_get_image_name classic)
-  mount_ubuntu_image "$NESTED_IMAGES_DIR/$IMAGE_NAME" "$IMAGE_MOUNTPOINT"
+    # install the snapd deb from spread so we are using the same version to 
+    # validate the seed as well as call preseed, etc. 
+    dpkg -i "$SPREAD_PATH"/../snapd_*.deb
 
-  # on 20.04 snapd from the deb is newer than snapd from seeds;
-  # this is not a sensible scenario for preseeding but since
-  # we're cheating and preseeding images that were not meant to be
-  # preseeded in their current state, we need to inject newer snapd
-  # into seeds/ to make snap-preseed and the test happy.
+    # create a VM and mount a cloud image
+    nested_create_classic_vm
+    mkdir -p "$IMAGE_MOUNTPOINT"
+    IMAGE_NAME=$(nested_get_image_name classic)
+    mount_ubuntu_image "$NESTED_IMAGES_DIR/$IMAGE_NAME" "$IMAGE_MOUNTPOINT"
 
-  # add snapd from this branch into the seed
-  repack_snapd_deb_into_snapd_snap "$PWD"
-  mv snapd-from-deb.snap snapd.snap
-  inject_snap_into_seed "$IMAGE_MOUNTPOINT" snapd
+    # on 20.04 snapd from the deb is newer than snapd from seeds;
+    # this is not a sensible scenario for preseeding but since
+    # we're cheating and preseeding images that were not meant to be
+    # preseeded in their current state, we need to inject newer snapd
+    # into seeds/ to make snap-preseed and the test happy.
 
-  # for images that are already preseeded, we need to undo the preseeding there
-  echo "Running preseed --reset for already preseeded cloud images"
-  SNAPD_DEBUG=1 /usr/lib/snapd/snap-preseed --reset "$IMAGE_MOUNTPOINT"
+    # add snapd from this branch into the seed
+    repack_snapd_deb_into_snapd_snap "$PWD"
+    mv snapd-from-deb.snap snapd.snap
+    inject_snap_into_seed "$IMAGE_MOUNTPOINT" snapd
+
+    # for images that are already preseeded, we need to undo the preseeding there
+    echo "Running preseed --reset for already preseeded cloud images"
+    SNAPD_DEBUG=1 /usr/lib/snapd/snap-preseed --reset "$IMAGE_MOUNTPOINT"
 
 restore: |
-  #shellcheck source=tests/lib/nested.sh
-  . "$TESTSLIB/nested.sh"
-  nested_destroy_vm
+    #shellcheck source=tests/lib/nested-tool.sh
```

Comment:
With both changes to nested and changes to indentation this is pretty hard to read.

Reply:
+1, this seems to contain changes that are not strictly related and change indentation but also re-shuffle the order of operations. Could you please propose this separately?